lib/main.js

```diff
@@ -27,11 +27,16 @@ const defaultOptions = {
   webpackDevConfig: require('./webpack-dev-config.js'),
   webpackDevOptions: { noInfo: true, publicPath: '/' },
   webpackDevBuildCallback: () => console.log('webpack dev build done.'),
+  indexSSR: false,
 };
 
 
 function middleware(options = {}) {
   options = _.merge(defaultOptions, options);
+  if (options.indexSSR) {
+    options.optsServeClient.index = false;
```

Comment:
This looks nice, but it will also stop sreving `client/foo/index.html` in addition to `client/index.html` which is only what we want to stop serving. Hm, but it's a rare case. I think it's OK.